src/router/console/accounts.js

```diff
@@ -22,15 +22,16 @@ export default [
         component: () => import('@/views/accounts/Account/index.vue'),
         meta: {
           title: i18n.t('AssetAccount'),
+          showInSearch: true,
           app: 'accounts',
           permissions: ['accounts.view_account']
         }
       },
       {
         path: ':id',
         component: () => import('@/views/accounts/Account/AccountDetail/index.vue'),
-        name: 'AssetAccountDetail',
-        meta: { title: i18n.t('AssetAccountDetail') },
+        name: 'AccountDetail',
+        meta: { title: i18n.t('AccountDetail') },
         hidden: true
       }
     ]
```

Comment:
There is no issue detected in the provided code snippet. However, there could be room for optimization in terms of documentation and naming conventions. The current code seems to follow good practices but may need slight modifications if more complex functionalities or changes are required later on.